### PR TITLE
Enabling new load xclbin flow for versal in sw emu

### PR DESCRIPTION
**Pull request by vboggara-xilinx**
Enabling new load xclbin flow for versal platforms in sw emu.

### CODE REVIEW

File: src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx [hunk 1]
```diff
@@ -80,7 +80,7 @@ namespace xclcpuemhal2 {
     return true;
   }
 
-  bool validateXclBin(const xclBin *header, std::string &xclBinName)
+  bool validateXclBin(const xclBin *header, std::string &xclBinName, bool &newFlow)
```
Comment (chvamshi-xilinx) [inline src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx:83]: Can you please give some name to newflow.

**Review by chvamshi-xilinx — COMMENTED** [review]
Other changes are looking good. Just give a meaningful name to newflow.

File: src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx (hunk 1 above)
Comment (vboggara-xilinx) [reply src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx]: @chvamshi-xilinx , comments addressed and submitted change again, please check.

**Comment by vboggara-xilinx** [comment]
Tested these changes for both aie and non aie flow with versal platforms.

**Pull request merged**